tests/views_integration/Tribe/Events/Views/V2/Views/__snapshots__/List_ViewTest__test_render_with_upcoming_events__1.php

```diff
@@ -25,11 +25,15 @@ class="tribe-events-view-loader tribe-common-a11y-hidden"
 	data-js="tribe-events-view-data"
 	type="application/json"
 >
+<<<<<<< HEAD
```

Comment:
@Camwyn this looks like a merge artifact.

Reply:
dangit